docker/buildkite-hosted-agent-bacalhau/Dockerfile

```diff
@@ -0,0 +1,43 @@
+FROM buildkite/hosted-agent-base
```

Comment:
Nice. Much cleaner that circleci setup. Can we put everything related to buildkite under `/buidlkite` directory, including this Dockerfile?